client/lib/signup/hint-data.js

```diff
@@ -1,673 +1,677 @@
+	Maps: i18n.translate( 'Maps' ),
+	Marketing: i18n.translate( 'Marketing' ),
+	'Marketing services': i18n.translate( 'Marketing services' ),
+	Massages: i18n.translate( 'Massages' ),
+	Mathematics: i18n.translate( 'Mathematics' ),
+	Mechanic: i18n.translate( 'Mechanic' ),
+	'Medial news': i18n.translate( 'Medial news' ),
+	Medical: i18n.translate( 'Medical' ),
+	'Medical technology': i18n.translate( 'Medical technology' ),
+	Medicine: i18n.translate( 'Medicine' ),
+	Meditation: i18n.translate( 'Meditation' ),
+	Men: i18n.translate( 'Men' ),
+	"Men's heatlh": i18n.translate( "Men's heatlh" ),
+	'Mental health': i18n.translate( 'Mental health' ),
+	'Mexican restaurant': i18n.translate( 'Mexican restaurant' ),
+	Military: i18n.translate( 'Military' ),
+	Militia: i18n.translate( 'Militia' ),
+	Milk: i18n.translate( 'Milk' ),
+	Minecraft: i18n.translate( 'Minecraft' ),
+	Ministry: i18n.translate( 'Ministry' ),
+	'Mobile app': i18n.translate( 'Mobile app' ),
+	'Mobile electronics': i18n.translate( 'Mobile electronics' ),
+	'Mommy blog': i18n.translate( 'Mommy blog' ),
+	Motivation: i18n.translate( 'Motivation' ),
+	'Motivational speaker': i18n.translate( 'Motivational speaker' ),
+	Motorhomes: i18n.translate( 'Motorhomes' ),
+	'Movie production': i18n.translate( 'Movie production' ),
+	Movies: i18n.translate( 'Movies' ),
+	'Moving service': i18n.translate( 'Moving service' ),
+	'Moving services': i18n.translate( 'Moving services' ),
+	Multimedia: i18n.translate( 'Multimedia' ),
+	'Multimedia law': i18n.translate( 'Multimedia law' ),
+	'Multimedia production': i18n.translate( 'Multimedia production' ),
+	Music: i18n.translate( 'Music' ),
+	'Music classes': i18n.translate( 'Music classes' ),
+	'Music lessons': i18n.translate( 'Music lessons' ),
+	'Music production': i18n.translate( 'Music production' ),
+	Musician: i18n.translate( 'Musician' ),
+	Mystery: i18n.translate( 'Mystery' ),
+	'Natural health products': i18n.translate( 'Natural health products' ),
+	'Natural health remedies': i18n.translate( 'Natural health remedies' ),
+	'Network marketing': i18n.translate( 'Network marketing' ),
+	Networking: i18n.translate( 'Networking' ),
+	News: i18n.translate( 'News' ),
+	Newspaper: i18n.translate( 'Newspaper' ),
+	'Non-profit': i18n.translate( 'Non-profit' ),
+	Novel: i18n.translate( 'Novel' ),
+	Nutrition: i18n.translate( 'Nutrition' ),
+	'Online business': i18n.translate( 'Online business' ),
+	'Online store': i18n.translate( 'Online store' ),
+	Optician: i18n.translate( 'Optician' ),
+	Optomitrist: i18n.translate( 'Optomitrist' ),
+	'Organic farming': i18n.translate( 'Organic farming' ),
+	Organization: i18n.translate( 'Organization' ),
+	Outdoors: i18n.translate( 'Outdoors' ),
+	Outfits: i18n.translate( 'Outfits' ),
+	Painter: i18n.translate( 'Painter' ),
+	Painting: i18n.translate( 'Painting' ),
+	Paranormal: i18n.translate( 'Paranormal' ),
+	Parenting: i18n.translate( 'Parenting' ),
+	Parties: i18n.translate( 'Parties' ),
```

Comment:
This could be ambiguous. I suppose it's about parties as in "birthday parties" but it could also be understood as in "Political Party". Maybe we should add the latter and change "Parties" to something like "Parties and Festivities"?

Reply:
Yeah, I like both your suggestions, it makes it less ambiguous. Let's do that.